client/network/src/transport.rs

```diff
@@ -116,8 +118,19 @@ pub fn build_transport(
 		.upgrade(upgrade::Version::V1Lazy)
 		.authenticate(authentication_config)
 		.multiplex(multiplexing_config)
-		.timeout(Duration::from_secs(20))
-		.boxed();
+		.timeout(Duration::from_secs(20));
 
-	(transport, bandwidth)
+	if !memory_only {
+		let webrtc_transport = WebRTCTransport::new(keypair);
```

Comment:
I don't actually know if this is the appropriate place, but the WebRTC certificate should be saved and loaded from disk. We already do this with the `keypair`, you should be able to follow the trail and find where we do it.